governance/communications/call-to-action.md

```diff
@@ -0,0 +1,20 @@
+# Call to Action Template
+
+## Security Enthusiasts!
+CNCF Security Tag is looking to embark on another tech security adventure!  [2 sentences explanation of what the thing is].
+[background about why this exists, about 2-3 sentences].  We'd like some help.
+ 
+## We want you!
+This is where you (yes you!) come in.  We're looking for community members and colleagues [review/contribute] to this [what the thing is called], tracked under [issue number and link to issue], so that we can provide the cloud native community with [what the intent is]!
+[what this means for the community]. We hope you can join us on this significant contribution opportunity. [If needed by a specific date, say so here].
+
+## Ready to dive in?  
+Comment on the issue [relink the issue] (#XXX) and join the [link to the slack channel] Slack channel!
+
+## References
+[Relink everything again here with what it is and the URLs blown out (not embedded)]
+
+Examples:
```

Comment:
Pls make list below bulleted list -- hard to read as is, since they show up as all one line (markdown need double-carriage-return to make new paragraph

So, formatting like this should work

```
Examples:
* https://lists.cncf.io/g/cncf-tag-security/message/58
* https://lists.cncf.io/g/cncf-tag-security/message/54
* https://lists.cncf.io/g/cncf-tag-security/message/53
```

Reply:
Thanks, done